test/fixtures/amp-bind-integrations/dynamic-tags.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-bind integration test</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-custom>
```

Comment:
Done!
Added commits for having a separate test file using this fixture as well.